src/nano/nano_signal_slot.hpp

```diff
@@ -71,7 +71,7 @@ class Signal<RT(Args...)> : private Observer
 
   template <typename L>
   void disconnect(L* instance) {
-    Observer::remove(Delegate::template bind(instance), this);
+    Observer::remove(Delegate::bind(instance), this);
```

Comment:
Clang has a new warning `-Wmissing-template-arg-list-after-template-kw` that requires a template argument list after using the template keyword.

See llvm/llvm-project#94194 for the upstream clang changes.

I'm using CTAD.